subsystem/sparkyShooterPivot.py

```diff
@@ -47,6 +49,10 @@ def getMeasurement(self):
     def setPostion(self, position: float):
         return
     
+    def setCurrentLimit(self, limitAmps: int = 20):
```

Comment:
move check for current to this call then users don't care and we can change how to track